src/utils/self_test/SConscript

```diff
@@ -9,24 +9,39 @@ import os
 import daos_build
 
 SELF_TEST = 'self_test.c'
+LIB_CONFIG_INI = 'src/configini.c'
+LIB_CONFIG_CNF = 'etc/selftest.cnf'
+LIB_CONFIG_ECNF = 'etc/selftest_expected.cnf'
+
+# Function to filter out gcc flags from this compilation env
+def remove_certain_gcc_flags(str):
```

Comment:
We're not allowed to use [`str`](https://docs.python.org/3/library/functions.html#func-str), as it's a Python keyword.  Search and replace the use of this `str` to something that's a non-keyword in Python, e.g., `_str`.

Reply:
Changed name to _str